pysat/_meta.py

```diff
@@ -478,6 +479,7 @@ def __setitem__(self, data_vars, input_dat):
             # from a Meta object. Set data using standard assignment via a dict
             in_dict = input_data.to_dict()
             if 'children' in in_dict:
+                self.warn_meta_children()
```

Comment:
From inspection of the tests, this is only raised when using the `meta.rename` function.  However, this occurs after higher order meta is set, so I don't think this is needed.

Reply:
keep this line, but add test